.github/fluent_linter_config.yml

```diff
@@ -25,3 +25,8 @@ CO01:
     exclusions:
         files: []
         messages: []
+VC:
+    disabled: false
+    exclusions:
```

Comment:
The rule is only enabled/disabled, there are no exceptions.

```
VC:
    disabled: false
```

Is all you need.